policies/api/http/endpoint_test.go

```diff
@@ -40,11 +40,15 @@ visor:
       type: pcap
       config:
         iface: eth0`
-	limit        = 10
-	invalidToken = "invalid"
-	maxNameSize  = 1024
-	contentType  = "application/json"
-	wrongID      = "28ea82e7-0224-4798-a848-899a75cdc650"
+	limit                    = 10
+	invalidToken             = "invalid"
+	maxNameSize              = 1024
+	contentType              = "application/json"
+	wrongID                  = "28ea82e7-0224-4798-a848-899a75cdc650"
+	invalidJson              = "{"
+	validDatasetJson         = "{\n    \"name\": \"my-dataset-json\",\n    \"agent_group_id\": \"8fd6d12d-6a26-5d85-dc35-f9ba8f4d93db\",\n    \"agent_policy_id\": \"86b7b412-1b7f-f5bc-c78b-f79087d6e49b\",\n    \"sink_id\": \"f5b2d342-211d-a9ab-1233-63199a3fc16f\"\n,\n    \"tags\": {\n        \"region\": \"eu\",\n        \"node_type\": \"dns\"\n    }}"
+	validDatasetYaml         = `{"name": "my-dataset-yaml", "agent_group_id": "8fd6d12d-6a26-5d85-dc35-f9ba8f4d93db", "agent_policy_id": "86b7b412-1b7f-f5bc-c78b-f79087d6e49b", "sink_id": "urn:uuid:f5b2d342-211d-a9ab-1233-63199a3fc16f", "tags": {"region": "eu", "node_type": "dns"}}`
```

Comment:
I'm not sure if we have a yaml format when create a dataset. I think we just use the policy, agent groups and sinks ids... Please check it.